lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -233,6 +257,20 @@ switchport_trunk_native_vlan:
   config_set_append: "%s switchport trunk native vlan %s"
   default_value: 1
 
+vlan_mapping:
+  _exclude: [/N3/, /N5K/, /N6K/, /N9K/]
+  multiple:
+  config_get_token_append: '/^switchport vlan mapping (\d+) (\d+)/'
```

Comment:
Its not available on 7k, but it is there on 9k. 
But for now I'd suggest to put it in the rally backlog as I haven't seen any config using the 'inner' config